src/Sarif.Multitool/Rules/DSP1003.RegionsMustProvideRequiredProperties.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
     public class RegionsMustProvideRequiredProperties : SarifValidationSkimmerBase
```

Comment:
>RegionsMustProvideRequiredProperties [](start = 17, length = 36)

While we are at it: please rename this class, and this file, and all the associated test files, and the resource string names, to: `ProvideRequiredRegionProperties`.

I want the rule names to consistently have the form "Provide required X properties" instead of "X must provide required properties". We use the "Provide..." form almost everywhere but a couple were inconsistent.